kubetest/main.go

```diff
@@ -415,10 +417,22 @@ func complete(o *options) error {
 	return nil
 }
 
+func flushMem() {
```

Comment:
probably need to be in build.go? and only for bazel build?

Reply:
I moved it, but I think we should call it after any building, we should try hard to release memory before the actual e2e's since they need much less.